onnxruntime/contrib_ops/rocm/diffusion/group_norm_impl_kernel.cuh

```diff
@@ -0,0 +1,209 @@
+#include "hip/hip_runtime.h"
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 1993-2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
```cpp
#pragma once

// Modifications Copyright (c) Microsoft.
// Licensed under the MIT License.

...
#include "hip/hip_runtime.h"
```

Reply:
The kernel is copied from TRT, I think we can keep the older Copyright and add Copyright Microsoft .

Reply:
Done. Most of our file put Copyright at the top, I think we can follow the same.